mbed_host_tests/host_tests_conn_proxy/conn_proxy.py

```diff
@@ -231,10 +231,15 @@ def __send_sync(timeout=None):
         else:
             # Return if state machine in host_test_default has finished to end process
             if key == '__host_test_finished' and value == True:
-                logger.prn_inf("received special even '%s' value='%s', finishing"% (key, value))
+                logger.prn_inf("received special event '%s' value='%s', finishing"% (key, value))
                 connector.finish()
                 return 0
-            if not connector.write_kv(key, value):
+            elif key == '__reset_dut':
```

Comment:
Let's make this key match the event queue key: `__reset_dut` -> `__soft_reset_dut`.